### PR TITLE
Flushout sidebar

### PR DESCRIPTION
**Pull request by hueblerw**
I hope this changing the menu bar doesn't mess with anybodies branches.  Let me know if there are issues.  

Includes a crude implementation of the favorites page but I need data to test it with and will continue to work on setting up the controllers to handle said data.

Basically ignore commits 3 and 4 as I was forced to roll them back because they damaged the registration form.  Will continue to work on them.

Basically, wanted to throw these menu changes before we got too involved.

### CODE REVIEW

**Comment by romanjesus** [comment]
Looks good 👍
